models/ht_neuron.cpp

```diff
@@ -49,8 +49,7 @@ RecordablesMap< ht_neuron >::create()
     Name( "Theta" ), &ht_neuron::get_y_elem_< ht_neuron::State_::THETA > );
   insert_(
     Name( "g_AMPA" ), &ht_neuron::get_y_elem_< ht_neuron::State_::G_AMPA > );
-  insert_(
-    Name( "g_NMDA" ), &ht_neuron::get_y_elem_< ht_neuron::State_::G_NMDA > );
+  insert_( Name( "g_NMDA" ), &ht_neuron::get_g_NMDA_ );
```

Comment:
Why is the recordable ``g_NMDA`` different from the state variable? (cf. ``get_g_NMDA``)

**EDIT:** ok, I understand, the state variable ``G_NMDA`` is the g(t) from the docstring and the recordable is the real ``g_NMDA = g(t)*m(V, t)``, which cannot be stored directly as a state variable since it is not what's updated.
But then the naming looks misleading to me, I guess we should change ``G_NMDA`` to something else and update the docstring... though I must say I'm at a loss regarding the name. Maybe ``G_T`` would be the most sensible choice...